core/client/src/main/java/alluxio/client/file/options/OpenFileOptions.java

```diff
@@ -91,6 +120,16 @@ public OpenFileOptions setLocationPolicy(FileWriteLocationPolicy locationPolicy)
   }
 
   /**
+   * @param policy the block location policy for the UFS read
+   * @return the UFS read location policy
+   */
+  @JsonIgnore
```

Comment:
i tried to be consistent with setLocationPolicy right above. I think it is because it is hard to set a BlockLocationPolicy *object* from json as we have no json definitions for the policies.